cfme/cloud/instance/__init__.py

```diff
@@ -152,11 +152,17 @@ def is_displayed(self):
             relationship_provider_name = relationships.get_text_of('Cloud Provider')
         except (NameError, NoSuchElementException):
             logger.warning('No "Cloud Provider" Relationship, assume instance view not displayed')
-            raise NotImplementedError("This view has no unique markers for is_displayed check")
-        return (
-            self.in_cloud_instance and
-            self.entities.title.text == 'Instance "{}"'.format(expected_name) and
-            relationship_provider_name == expected_provider)
+            # for archived instances the relationship_provider_name is removed from the summary
+            # table
+            return (
+                self.in_cloud_instance and
+                self.entities.title.text == 'Instance "{}"'.format(expected_name)
+            )
+        else:
+            return (
```

Comment:
I think better to return this in `try` than `else`

Reply:
@digitronik @sshveta done